### ISSUE TITLE
During 'Parsing' check, Accessibility Insight injected styles are also reported

### ISSUE THREAD

**Issue by joppekroon** [microsoft/accessibility-insights-web#1553]
## Describe the bug
When running the 'Parsing' check, the checker reports failures with <style> elements that have an unnecessary type attribute. It took me a while to figure out that these were failures caused by injections from accessibility insights ([dev tools performance issue didn't help](https://github.com/microsoft/accessibility-insights-web/issues/1503)). 

**To Reproduce**
Run the Nu HTML Checker on the page where Accessibility Insights is active, the Checker will report several "The type attribute for the style element is not needed and should be omitted." warnings.

## Expected behavior

The 'Parsing' check can be run without being confused by parsing errors or warnings that do not belong to the page under test itself.

**Comment by ferBonnin** [comment]
Per conversation with Peter, needs dev investigation. Is there anything we need to fix on our side: If we can't hide the styles from the validator, lets fix the issue the validator is calling out.
Dan might have more context/knowledge on the validator since we investigated other issues with it in the past.

**Comment by karanbirsingh** [comment]
I have created an issue and PR [here](https://github.com/microsoft/rushstack/issues/1673) in `microsoft/rushstack/load-themed-styles` to suggest removing the type attribute from style tags. If the PR is accepted, then the warnings will subside once we consume a version of `office-ui-fabric-react` that consumes an updated version of `load-themed-styles`. Marking as blocked

**Comment by karanbirsingh** [comment]
Turns out we can consume the indirect dependency ourselves - issue fixed in PR #1954

**Comment by ferBonnin** [comment]
Hi @joppekroon this issue is now fixed in our [Canary](https://chrome.google.com/webstore/detail/hbcplehnakffdldhldncjlnbpfgogbem) extension. 

Please let us know if this fixes the issue reported.

**Comment by joppekroon** [comment]
Yep, the injected styles in Canary no longer add the unnecessary type attribute. So the parsing check now only shows your own failures! ;-). 

This fixes the reported issue adequately.
